### PR TITLE
CONCF-191 enable new GlobalFooter on Oasis

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss [hunk 1]
```diff
@@ -1,113 +1,98 @@
...
-			display: inline-block;
-			margin-top: 8px;
-			padding: 0 8px;
+		margin: 0 101px 30px;
+		padding: 0 10px;
 
-			&:first-child {
-				border: 0;
-			}
+		@media #{$large-range} {
```
Comment (vforge) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: Please move this to root of the SCSS.

File: extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss [hunk 2]
```diff
@@ -1,113 +1,98 @@
...
-	font-size: 11px;
-	padding-bottom: 10px;
-
-	nav {
-		@include clearfix;
-		margin-top: 15px;
-		position: relative;
+	@include box-sizing(border-box);
+
+	*, *::after {
```
Comment (hakubo) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: How about making this global? or don't we already do it globally?

**Comment by hakubo** [comment]
Please wait with merge till Monday, so we have enough time for review

File: extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss (hunk 2 above)
Comment (vforge) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: @hakubo We're doing it for Venus skin only globally. It's neat idea to do this by default to Oasis too, tho.

File: extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss (hunk 1 above)
Comment (bognix) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: SASS is compiling this at root position:
```
    @media only screen and (min-width: 1371px) {
      .global-footer ul {
        margin: 0 113px 30px;
        padding: 0 12px; } }
    @media only screen and (max-width: 1023px) {
      .global-footer ul {
        margin: 0 77px 30px;
        padding: 0 8px; } }

Comment (vforge) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: :+1:

File: extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss [hunk 3]
```diff
@@ -1,113 +1,90 @@
...
-	box-sizing: content-box;
-	clear: left;
-	font-size: 11px;
-	padding-bottom: 10px;
-
-	nav {
-		@include clearfix;
-		margin-top: 15px;
-		position: relative;
+	@include box-sizing(border-box);
```
Comment (vforge) [inline extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss]: According to http://caniuse.com/#feat=css3-boxsizing ```box-sizing``` works everywhere - you can safely remove include.

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 4]
```diff
@@ -29,15 +45,21 @@ private function getGlobalFooterLinks() {
...
-		$catId = WikiFactoryHub::getInstance()->getCategoryId( $wgCityId );
-		$memcKey = wfMemcKey( self::MEMC_KEY_GLOBAL_FOOTER_LINKS, $wgContLang->getCode(), $wgLang->getCode(), $catId );
+		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $wgCityId );
+		$memcKey = wfSharedMemcKey(
+			self::MEMC_KEY_GLOBAL_FOOTER_LINKS,
+			$wgContLang->getCode(),
+			$wgLang->getCode(),
+			$verticalId,
+			self::MEMC_KEY_GLOBAL_FOOTER_VERSION
+		);
```
Comment (macbre) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: Thank you :)

For future comparison:

```
site> select bucket, key, sum from Mediawiki_App_Memcache_Keys_Stats_60m where time > now() - 3h and key =~ /mGlobalFooterLinks:en/ and bucket =~ /miss|hit/
┌──────────────────┬─────────────────┬────────┬─────────────────────────────────┬───────┐
│       time       │ sequence_number │ bucket │ key                             │ sum   │
├──────────────────┼─────────────────┼────────┼─────────────────────────────────┼───────┤
│ 10/3/15 09:00:00 │ 2346            │ miss   │ *:mGlobalFooterLinks:en:vi:*    │ 3     │
│ 10/3/15 09:00:00 │ 2345            │ miss   │ *:mGlobalFooterLinks:en:th:*    │ 1     │
│ 10/3/15 09:00:00 │ 2344            │ miss   │ *:mGlobalFooterLinks:en:sv:*    │ 1     │
│ 10/3/15 09:00:00 │ 2343            │ miss   │ *:mGlobalFooterLinks:en:ru:*    │ 6     │
│ 10/3/15 09:00:00 │ 2342            │ miss   │ *:mGlobalFooterLinks:en:ro:*    │ 1     │
│ 10/3/15 09:00:00 │ 2341            │ miss   │ *:mGlobalFooterLinks:en:pt-br:* │ 1     │
│ 10/3/15 09:00:00 │ 2340            │ miss   │ *:mGlobalFooterLinks:en:pl:*    │ 4     │
│ 10/3/15 09:00:00 │ 2339            │ miss   │ *:mGlobalFooterLinks:en:it:*    │ 3     │
│ 10/3/15 09:00:00 │ 2338            │ miss   │ *:mGlobalFooterLinks:en:es:*    │ 1     │
│ 10/3/15 09:00:00 │ 2337            │ miss   │ *:mGlobalFooterLinks:en:en:*    │ 26581 │
│ 10/3/15 09:00:00 │ 2336            │ miss   │ *:mGlobalFooterLinks:en:*:*     │ 4     │
└──────────────────┴─────────────────┴────────┴─────────────────────────────────┴───────┘
Query took  2364 ms
```

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 5]
```diff
@@ -29,15 +45,21 @@ private function getGlobalFooterLinks() {
...
 
 		$globalFooterLinks = $wgMemc->get( $memcKey );
```
Comment (macbre) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: Do we set the key anywhere? :)

**Pull request merged**